Dynatrace/dynatrace-operator
# Removes dynakube namespace from ignored namespaces

# Description

Removes the dynakube's namespace from the list of ignored namespaces.
Instead, the DTO components have now an annotation to not inject into them.
Only the ActiveGate does not so the Agent can inject into it if deployed.

## How can this be tested?

* Deploy a cluster
* Deploy a dynakube with cloud native and activegate
* Check that os agent pods, operator pod, csi pods and webhook pods have inject: false annotation
* Check that agent was injected into activegate container

## Checklist
